### DISCUSSION TITLE
Laravel Nova 4 Installation

### DISCUSSION THREAD

**Discussion by sagadsalem** [laravel/nova-issues#4019] [Q&A] [answered]
- Laravel Version: ^9.2
- Nova Version: ~4.0
- PHP Version: 8.1.4

Hi there

I just purchase a new Laravel nova 4 license and try to install it via composer
so I put the Laravel nova account credentials to the auth.json file and run ``` Composer update ```
I get this
```

**********************************************
Invalid username/password combination given.
**********************************************
    Failed to download Laravel/nova from dist: Invalid credentials for 'https://nova.laravel.com/dist/laravel/nova/laravel-nova-*******f.zip', aborting. Now trying to download from the source
```

and then ask me to get a Github access token so I create a new one

now I get this

```
Failed to execute git clone --mirror 'https://MY-GITHUB-ACCESS-TOKEM:***@github.com/laravel/nova.git'
  '/Users/mohammed/.composer/cache/vcs/git-github.com-laravel-nova.git/'

```

**Comment by ERPedersen** [comment, marked as answer]
**Make sure you use your license key instead of your password when prompted for a password by composer.**

I mistakingly entered my password, since that's have I've done on previous versions, and was met with the following message:

```
******************************************************************************
Unable to find Nova (Silver Surfer) license holder with the given license key.
******************************************************************************
```

Entering the license key instead of my password as described worked:

> When running composer update, you will be prompted to provide your Nova website username and [license key (opens new window)](https://nova.laravel.com/licenses), which should be used in place of your password. These credentials will authenticate your Composer session as having permission to download the Nova source code.
>
> https://nova.laravel.com/docs/4.0/installation.html

**Reply by sagadsalem** [reply]
thanks works just fine
